core/io/marshalls.cpp

```diff
@@ -641,6 +641,7 @@ Error decode_variant(Variant& r_variant,const uint8_t *p_buffer, int p_len,int *
 
 			ERR_FAIL_COND_V(len<4,ERR_INVALID_DATA);
 			uint32_t count = decode_uint32(buf);
+			// count always > 0
```

Comment:
Is the `ERR_FAIL_COND_V` test useful at all? `count` is a `uint32_t`, so it can't be < 0, right?